### PR TITLE
Typos, Markdown cleanup, and clarifications

### PR DESCRIPTION
**Pull request by linuxwolf**
based on work started by @rnewman, some editorial cleanup of the sync architecture.

### CODE REVIEW

**Comment by linuxwolf** [comment]
@pdehaan @rnewman 

Moved around the "themes" for better context.  If you all are ok with it, editorially, we'll merge this sometime tomorrow (Americas).

File: architecture/sync.md [hunk 1]
```diff
@@ -296,10 +277,7 @@ The steps to resolve keystore conflicts are as follows:
 
 #### Item Conflicts
 
-Reconciling conflicting item changes start with the following basic rules:
-
-* _Favor "update" actions over "remove" actions_. If an item is both removed and updated, treat it as updated.
-* _Favor local changes over remote changes_. For a given change within an item, favor the local change over the remote change.
+Reconciling conflicting changes within an item adheres to the basic theme **_favor `local` changes over `remote` changes_**.  The rationale is that any changes made on a device trigger an immediate sync, and therefore are more likely to match the user's current intent.
 
 Item conflicts are reconciled as follows:
```
Comment (pdehaan) [inline architecture/sync.md]: Uber-nit: Bulleted list below is funny in that it starts at "2." (in both the Markdown and rendered output).

**Comment by pdehaan** [comment]
LGTM, but I'll leave the final 👍 for @rnewman (or if @m8ttyB wants to give a quick 👀).

File: architecture/sync.md (hunk 1 above)
Comment (linuxwolf) [reply architecture/sync.md]: My jekyll render (which is what's used to get https://mozilla-lockbox.github.io/architecture/sync.html) looks correct to me; maybe it's something with Github's auto-render?

I can correct the numbering in the text, though.  That's a hold-over from a slew of edits.

File: architecture/sync.md [hunk 2]
```diff
@@ -11,21 +11,21 @@ Cloud-based synchronization can provide a user reliable access to their logins.
...
 * `remote` - An item or keystore whose representation changed in remote storage compared to the current stable state.
-* `local` - An item or keystore whose representation change in local storage compared to the current stable state..
+* `local` - An item or keystore whose representation changed in local storage compared to the current stable state.
 * `working` - An item or keystore in the process of conflict reconciliation.
 
 The following terms are also used:
 
-* `item` - The representation of a Lockbox entry (i.e., stored login credentials), realized as a JSON object.
+* `item` - The representation of a Lockbox entry (_e.g._, stored login credentials), realized as a JSON object.
 * `keystore` - The representation of Lockbox's item keystore, realized as a JSON objet.
```
Comment (rnewman) [inline architecture/sync.md]: objet -> object

File: architecture/sync.md [hunk 3]
```diff
@@ -160,22 +152,18 @@ When a local change is made to an item, the follow is performed:
...
 
 1. Verify authorization to remote storage service
-
     - **NOTE**: If this step fails, an error is propagated and the sync operation is terminated.
 
 2. Fetch remote changes
 3. Examine pending changes (and apply non-conflicting remote changes to stable)
 4. Reconcile any "update" conflicts (and treat as local changes)
-
     - **NOTE**: this step is skipped if the datastore is locked
```
Comment (rnewman) [inline architecture/sync.md]: Closing period.

File: architecture/sync.md [hunk 4]
```diff
@@ -190,15 +178,13 @@ Verification is performed as follows:
 
 1. The cached access token is examined; access token is missing or expired, an attempt is made to refresh it.
```
Comment (rnewman) [inline architecture/sync.md]: "if the" access token…

**Comment by linuxwolf** [comment]
@rnewman @pdehaan thanks for the 👀 !

**Pull request merged**